etc/kayobe/kolla/globals.yml

```diff
@@ -54,6 +54,8 @@ manila_tag: 7.0.2.1-1
 mariadb_tag: 7.0.2.1-1
 memcached_tag: 7.0.2.1-1
 monasca_tag: 7.1.0.1
+monasca_api_tag: 7.0.2.3
```

Comment:
I think we should set `kolla_source_version: refs/tags/stackhpc/7.0.2.3`, `kolla_openstack_release: 7.0.2.3-1` and this tag to `7.0.2.3-1` so that this PR contains all information required to apply it.

The `-1` postfix can then be bumped if you update the branches that you are working on to apply patches to these images.

Reply:
Still outstanding

Reply:
I've added the postfix, this is `outdated`

Reply:
The first bit though about the kolla vars

Reply:
Still missing kolla_source_version

Reply:
Okay... updated. I assumed the version bumping was arbitrary but this makes sense...